### PR TITLE
Remove stray 1px border from last post in topic

### PR DESCRIPTION
**Pull request by lukegb**

### CODE REVIEW

**Comment by tgxworld** [comment]
Hi Luke, could you provide some before and after screenshots to make it easier to review?

File: desktop/desktop.scss [hunk 1]
```diff
@@ -77,6 +77,11 @@
     }
 }
 
+//remove stray border from bottom of last post
+.topic-status-info {
+    border-top: 0;
```
Comment (tgxworld) [inline desktop/desktop.scss]: We prefer 2 spaces for indentation.

Comment (lukegb) [inline desktop/desktop.scss]: I'm confused - I looked at the lines above and they're 4 spaced (which is why I assumed the indentation standard was 4 spaces), but at the top they're 2 spaced.

Comment (tgxworld) [inline desktop/desktop.scss]: Looks like there is alot of copy-pasta code in there. 4 spaces are fine for now. I'll clean up the code soon. :+1:

**Comment by SamSaffron** [comment]
@rewphus what should we do here?

**Comment by lukegb** [comment]
Yikes, sorry about this. I'll grab some screenshots, if that's all that's holding this up.

**Pull request merged**